.environment/githooks.sh

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env bash
+
```

Comment:
Seeing some errors with this script on my Mac:

```bash
.environment git:(td/gitleaks) ./githooks.sh install --whatif 
./githooks.sh: line 8: declare: -A: invalid option
declare: usage: declare [-afFirtx] [-p] [name[=value] ...]
./githooks.sh: line 9: pre-commit.hook.sh: syntax error: invalid arithmetic operator (error token is ".hook.sh")
> Setting up your git hooks
```

```bash
.environment git:(td/gitleaks) ./githooks.sh install         
./githooks.sh: line 8: declare: -A: invalid option
declare: usage: declare [-afFirtx] [-p] [name[=value] ...]
./githooks.sh: line 9: pre-commit.hook.sh: syntax error: invalid arithmetic operator (error token is ".hook.sh")
> Setting up your git hooks

```

Reply:
Good catch, bash 3.* does not have support for associative arrays; I've pushed up 977b89e6b346ba9a5b6c0a27a0a82ad4f6b62792 and a731ef8a19a5727465898ec02d36fb23d48c2f49 to fix that.